packages/contentstack-export-to-csv/src/util/index.js

```diff
@@ -6,10 +6,9 @@ const fastcsv = require('fast-csv');
 const inquirer = require('inquirer');
 const debug = require('debug')('export-to-csv');
 const checkboxPlus = require('inquirer-checkbox-plus-prompt');
-
 const config = require('./config.js');
-const { cliux, configHandler } = require('@contentstack/cli-utilities');
-
+const { cliux, configHandler, HttpClient } = require('@contentstack/cli-utilities');
+const {cloneDeep} = require('lodash');
```

Comment:
Can we do a specific import to improve the import cost.?

`const cloneDeep = require('lodash/cloneDeep');`

Reply:
Yes used